packages/shared/src/components/cards/v1/FeedItemContainer.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {
+  forwardRef,
+  HTMLAttributes,
+  ReactElement,
+  ReactNode,
+  Ref,
+} from 'react';
+import { Post } from '../../../graphql/posts';
+import { Card } from './Card';
+import { RaisedLabel, RaisedLabelType } from './RaisedLabel';
+import { useFeedPreviewMode } from '../../../hooks';
+import { TypeLabel } from './TypeLabel';
+
+interface FeedItemContainerProps {
+  flagProps?: FlagProps;
+  children: ReactNode;
+  domProps: HTMLAttributes<HTMLDivElement>;
+}
+
+interface FlagProps extends Pick<Post, 'pinnedAt' | 'trending' | 'type'> {
+  adAttribution?: ReactElement | string;
+}
+
+function FeedItemContainer(
+  { flagProps, children, domProps }: FeedItemContainerProps,
+  ref?: Ref<HTMLElement>,
+): ReactElement {
+  const { adAttribution, pinnedAt, trending, type } = flagProps;
+
+  const raisedLabelType = pinnedAt
+    ? RaisedLabelType.Pinned
+    : RaisedLabelType.Hot;
+  const description =
+    raisedLabelType === RaisedLabelType.Hot && trending > 0
+      ? `${trending} devs read it last hour`
+      : undefined;
+  const isFeedPreview = useFeedPreviewMode();
+  const showFlag = (!!pinnedAt || !!trending) && !isFeedPreview && description;
+  const showTypeLabel = !!adAttribution || !!type;
+
+  return (
+    <Card
+      {...domProps}
+      data-testid="postItem"
+      ref={ref}
+      className={domProps?.className}
+    >
+      <fieldset>
```

Comment:
Semantically speaking, isn't `fieldset` used in a form?

Reply:
I would tend to agree. Was following the task definition on the issue here: https://dailydotdev.atlassian.net/browse/WT-2004

Reply:
I see. Let's probably seek opinion of the others as well @dailydotdev/web-team

Reply:
Fieldset can be used outside a form as well, though it's intend is to group form elements together. Is that the best practice, not sure at this moment, we shouldn't really be using it only from design perspective most probably, that's why it's a recommendation rather then a set way on how to do it, so if we feel it's a bad practice, let's design with normal divs.